echopype/convert/set_groups_ek80.py

```diff
@@ -388,6 +377,7 @@ def set_platform(self) -> xr.Dataset:
                         "axis": "T",
                         "long_name": "Timestamps for NMEA datagrams",
                         "standard_name": "time",
+                        "comment": "Time coordinate corresponding to GPS location.",
```

Comment:
Replace these 3 lines with what's used in ek60:

```python
                    {
                        **self._varattrs["platform_coord_default"]["time1"],
                        "comment": "Time coordinate corresponding to GPS location.",
                    },
```

The result is the same, but the code ends up being more consistent and leveraging 1.0.yml